File: sources/None/none/module/Category.java
package none.module;

public enum Category {
	
	COMBAT("Combat"),
	MOVEMENT("Movement"),
	PLAYER("Player"),
	RENDER("Render"),
	WORLD("World");
	
	private String name;

    Category(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return name;
    }
	
}

File: sources/Leaf devEnv/com/leafclient/leaf/management/mod/category/Category.kt
package com.leafclient.leaf.management.mod.category

import com.leafclient.leaf.management.utils.Labelable
import java.awt.Color

enum class Category(override val label: String, val color: Color): Labelable {

    FIGHT("Fight", Color(200, 40, 0)),
    VISUAL("Visual", Color(220, 220, 40)),
    MOVEMENT("Movement", Color(40, 120, 250)),
    WORLD("World", Color(40, 160, 40))

}